setup.py

```diff
@@ -75,13 +75,15 @@
         'libsass==0.14.2',  # 0.14.3 upwards changes imports for .css
         'lxml>=3.4.1',
         'mammoth>=1.4.4',
+        'pandas>=1.1.4',
         'psycopg2==2.7.3.2',  # 2.7.4 began using psycopg2-binary
         'requests>=2',
         'unicodecsv>=0.14.1',
         'whitenoise<2,>=1.0.6',
         'django-contrib-comments>=1.9.1',
         'XlsxWriter>=1.2.6',
         'xmldiff>=2.4',
+        'xlrd>=1.2.0',
```

Comment:
Is this used?

Reply:
tests failed to run without it?

Reply:
Huh. If you can determine it's a test dependency, then it should go under that section. Otherwise, it's a bit odd that it has showed up now (maybe pandas depends on it, although I think not?)